modules/apigee_edge_apidocs/tests/src/Kernel/ApiDocTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\content_entity_example\Kernel;
+
+use Drupal\apigee_edge_apidocs\Entity\ApiDoc;
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Test basic CRUD operations for our ApiDoc entity type.
+ *
+ * @group apigee_edge
+ * @group apigee_edge_apidocs
+ */
+class ApiDocTest extends KernelTestBase {
```

Comment:
Does this test provide any value if you test the same functionality on the UI within `ApiDocsAdminTest`?

Reply:
It can be a bit nicer to have a fast kernel test to fail quickly. Do you care about that?